### PR TITLE
allow model param for rerank

### PR DESCRIPTION
**Pull request by lfayoux**
We're going to be deploying multiple rerank models soon, sdk needs to allow for models to be passed in for rerank.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere',
-                 version='3.7.0',
+                 version='3.7.1',
```
Comment (sanderland) [inline setup.py:27]: nit: this is not a bugfix, so use 3.8

**Review by sanderland — APPROVED** [review]

**Review by abdullahkady — APPROVED** [review]

File: cohere/client.py [hunk 2]
```diff
@@ -403,12 +403,14 @@ def feedback(self, id: str, good_response: bool, desired_response: str = "", fee
...
                query: str,
                documents: Union[List[str], List[Dict[str, Any]]],
+               model: str = None,
                top_n: Optional[int] = None) -> Reranking:
         """Returns an ordered list of documents ordered by their relevance to the provided query
 
         Args:
             query (str): The search query
             documents (list[str], list[dict]): The documents to rerank
+            model (str): (Optional) The model to use for re-ranking
```
Comment (co-sylvie) [inline cohere/client.py:413]: could we say what the defaults are for these args in the doc string?

Comment (lfayoux) [reply cohere/client.py:413]: the default is none

Comment (co-sylvie) [reply cohere/client.py:413]: i mean the default behavior, if it's none we default to the english model right?

Comment (lfayoux) [reply cohere/client.py:413]: yeah but the default model could change, it will be set in the docs once this goes public

**Review by co-sylvie — APPROVED** [review]
✨

**Pull request merged**